client/src/components/AddCard/AddCardInfo/AddCardInfo.tsx

```diff
@@ -0,0 +1,31 @@
+import { Elements } from '@stripe/react-stripe-js';
+import { loadStripe } from '@stripe/stripe-js';
+import AddCardInfoForm from './AddCardInfoForm/AddCardInfoForm';
+
+/*interface Props {
+  clientSecret: string;
+}*/
+
+const AddCardInfo = (props: any): JSX.Element => {
+  const options = {
+    // passing the client secret obtained in step 2
```

Comment:
Let's get rid of this comment

Reply:
This part of Stripe is new to me. So I keep it for reference. I am sorry to have so many comments here. 👍 Done